LSP/app/automatic_decision_analyst.rb

```diff
@@ -0,0 +1,11 @@
+require_relative 'analyst'
+
+class AutomaticDecisionAnalyst < Analyst
+  def analyse(application, reject)
+    if reject
+      puts "#{name} approved #{application.customer_name}'s application!"
+    else
+      puts "#{name} rejected #{application.customer_name}'s application."
```

Comment:
Essa aqui é a parte que eu falei que eu tinha invertido no enunciado do desafio, desculpa :see_no_evil: 

Aqui deveria ficar assim, né?:
```suggestion
    if reject
      puts "#{name} rejected #{application.customer_name}'s application."
    else
      puts "#{name} approved #{application.customer_name}'s application!"
```

Quando reject for true, a application é rejeitada. Quando for false, é aprovada.

Reply:
Eitaaaaa, eu achei que tivesse alterado...mas recomecei e não me atentei xD